bin/cml-send-comment.js

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env node
 
+const print = console.log;
 console.log = console.error;
```

Comment:
really? why?

Reply:
You're 27 days late: https://github.com/iterative/cml/issues/22#issuecomment-846022553 😈  We use this on all the commands and changing it would be out of scope for this pull request, but it would be nice to choose a less "interesting" approach.

Reply:
:upside_down_face: